### ISSUE TITLE
add proper citations to existing articles

### ISSUE THREAD

**Issue by maxheld83** [subugoe/oauni#29]
the article version of the preprint currently only has hard-copied citations; they're not run through pandoc citeproc and are not included in the `*.bib`.

**Comment by maxheld83** [comment]
@Ahobert @njahn82 this is just a small thing, but do you by any chance have a `*.bib` for the references in the preprint that you could send me or commit?
The hard-copied citations in the vignette are bothering me 🤓

**Comment by Ahobert** [comment]
good point. I don't think there is a bib-file so far, but we do have a Zotero collection for it. Let me quickly check if that really is complete and corresponds to the Preprint bibliography, then I'll generate a bib-file from it.

**Comment by maxheld83** [comment]
Zotero has great (even automatic) `*.bib` export, so that should work great.
